aten/src/ATen/native/DispatchStub.cpp

```diff
@@ -24,20 +25,20 @@ static inline bool cpu_has_vxe()
 #endif
 
 static CPUCapability compute_cpu_capability() {
-  auto envar = std::getenv("ATEN_CPU_CAPABILITY");
-  if (envar) {
+  const auto envar = c10::utils::get_env("ATEN_CPU_CAPABILITY");
```

Comment:
I think this would be dramatically safer by changing the API to take a default value and then not returning a std::optional